src/CMakeLists.txt

```diff
@@ -693,28 +710,27 @@ list(APPEND TINYSPLINE_CXX_SOURCE_FILES
 include(CMakePackageConfigHelpers)
 
 # C library
-add_library(tinyspline_shared SHARED ${TINYSPLINE_C_SOURCE_FILES})
-set_target_properties(tinyspline_shared PROPERTIES
+add_library(tinyspline ${TINYSPLINE_C_SOURCE_FILES})
+if(NOT BUILD_SHARED_LIBS)
+    target_compile_definitions(tinyspline PUBLIC TINYSPLINE_STATIC)
```

Comment:
I don't understand what the purpose of this line is. Is it required by ´target_link_libraries´ in the examples? What is `TINYSPLINE_STATIC`

Reply:
spaces -> tabs